vars/pipelineCapsuleSystemTests.groovy

```diff
@@ -72,6 +72,11 @@ void call() {
 
   pipeline {
     agent none
+    options {
+      timestamps()
+      ansiColor('x-term')
+      timeout(time: 3, unit: 'HOURS')
```

Comment:
We cannot do that here. because some of the runs are designed to run for a few days